includes/ziti/ziti.h

```diff
@@ -671,6 +671,9 @@ extern const char *ziti_conn_source_identity(ziti_connection conn);
 ZITI_FUNC
 extern int ziti_service_available(ziti_context ztx, const char *service, ziti_service_cb cb, void *ctx);
 
+ZITI_FUNC
+extern const ziti_service *ziti_service_for_addr(ziti_context ztx, ziti_protocol proto, const char *addr, int port);
```

Comment:
It would be really, really nice if there was a version of this function that takes a numeric (non-string) IP.

I'm thinking of the tSDK, which will be calling this every time it reads a SYN packet or UDP packet with an unfamiliar src ip:port. At that point, the only thing the tSDK has is an `ip_addr_t`, and it would be unfortunate to convert that to string for calling this function. (especially since the string will be parsed back into numeric form by `ziti_intercept_match`/`ziti_address_match_list`).

`ip_addr_t` would be ideal from the tSDK's perspective, but I appreciate that we don't want ziti-sdk-c depending on LWIP. `ziti_address *` seems like a nice compromise, since the tSDK can efficiently create one from an `ip_addr_t`.

Reply:
added

Reply:
The "reverse lookup" is actually `model_map_getl` that uses the numeric IP as the key. So all of the intercept lookups currently do not involve parsing.